oracle/src/confirmRelay.js

```diff
@@ -153,11 +153,11 @@ async function main({ sendJob, txHashes }) {
 }
 
 async function sendJobTx(jobs) {
-  await GasPrice.start(chain, true)
-  const gasPrice = GasPrice.getPrice().toString(10)
-
   const { web3 } = config.sender === 'foreign' ? config.foreign : config.home
 
+  await GasPrice.start(chain, web3, true)
```

Comment:
Will `mycrypto/gas-estimation` work if specify several URLs in the RPC URL parameter?

Reply:
It should, all RPC calls are going through the same created web3 object, so any url fallback logic should work here as well.